tests/conftest.py

```diff
@@ -484,24 +485,27 @@ def no_write_perm_path(tmp_path_factory) -> str:
 
 
 @pytest.fixture(scope="session")
-def balsamic_cache(tmp_path_factory, reference):
-    """Create and return the path for balsamic-cache"""
-    cache_dir = tmp_path_factory.mktemp("balsmic_cache")
+def references_dir(test_data_dir) -> Path:
+    """Return a references directory path."""
+    return Path(test_data_dir, "references")
 
-    cache_container = cache_dir / balsamic_version / "containers" / "align_qc"
-    cache_container.mkdir(parents=True, exist_ok=True)
-    cache_container_example = cache_container / "example.sif"
-    cache_container_example.touch()
 
-    cache_reference = cache_dir / balsamic_version / "hg19"
-    cache_reference.mkdir(parents=True, exist_ok=True)
-
-    cache_reference_json = cache_reference / f"reference.{FileType.JSON}"
-    cache_reference_json.touch()
-    with open(cache_reference_json, "w") as fp:
-        json.dump(reference, fp)
-
-    return cache_dir.as_posix()
+@pytest.fixture(scope="session")
+def balsamic_cache(
+    tmp_path_factory: TempPathFactory, reference: Dict[str, Path], references_dir: Path
+) -> str:
+    """Create and return the path for balsamic-cache"""
```

Comment:
Geez, so sloppy... 🙄 
```suggestion
    """Create and return the path for balsamic-cache."""
```